### ISSUE TITLE
enf.cacert.pem is overwritten but not fully replaced

### ISSUE THREAD

**Issue by dberliner** [xaptum/enftun#100]
/etc/enftun/enf.cacert.pem is overridden to provide the default CA cert, but the operation leaves any trailing file data intact. Below is an example of what may be left on the machine

```
-----BEGIN CERTIFICATE-----
MIIBJzCBzwIIYUquXYZXXdswCgYIKoZIzj0EAwIwGzEZMBcGA1UEAwwQWEFQVFVN
Uk9PVDMzNTgxNzAiGA8wMDAwMDEwMTAwMDAwMFoYDzk5OTkxMjMxMjM1OTU5WjAb
MRkwFwYDVQQDDBBYQVBUVU1ST09UMzM1ODE3MFkwEwYHKoZIzj0CAQYIKoZIzj0D
AQcDQgAEVld7eTb+BOcTex9GBqZ9hr3eRhIjuprj5igLXb+F/ANnRY6yvDvYzZCt
2wRBivtW4Rpsi0SEEafTHuRWQTjq6jAKBggqhkjOPQQDAgNHADBEAiBiBwVHgHOq
ncWZEqZq31lQ4Y9Ks622arJuncMk5XHcMwIgJzuqN5MTpeWGBQ+RrvzB/zBlzrl5
ob2zZNkZ6sfIa8s=
-----END CERTIFICATE-----

BAMCA0gAMEUCIDT1nEl1
mYF9PWMEHFku4q9bupVrjpTkpBuES3t+7U6pAiEA214UME1YWo/EH9sS1mhVCRxD
HhpBG9YdhAuFk9bBSu0=
-----END CERTIFICATE-----
```

**Comment by drbild** [comment]
Reported as a bug in upstream `xtt` library: https://github.com/xaptum/xtt/issues/117

Keeping open here until a new version of `xtt` with a fix is released.

**Comment by drbild** [comment]
Fixed by xaptum/xtt#118
